mods/ascent/machines/ship_machines.dm

```diff
@@ -127,30 +98,6 @@ MANTIDIFY(/obj/item/chems/chem_disp_cartridge, "canister", "chemical storage")
 		/decl/stock_part_preset/radio/event_transmitter/airlock/shuttle = 1
 	)
 
-/obj/machinery/power/apc/hyper/ascent/north
```

Comment:
These appear to be removed and not replaced, why is that?

Reply:
Didn't really seem to serve much flavor/purpose anymore. I want to add it in the future with better features.